FSharpLu.Json/Compact.fs

```diff
@@ -10,18 +13,33 @@ open Microsoft.FSharp.Reflection
 type CompactUnionJsonConverter() =
     inherit Newtonsoft.Json.JsonConverter()
 
+    let processWithContractResolver (serializer:JsonSerializer) (name:string) =
```

Comment:
In order to be inlined it would need to be moved outside of the type. Not sure if it's worth it...